### PR TITLE
fix: add Aniruddha Inge

### PR DESCRIPTION
**Pull request by Aniruddha-Inge**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/864"></a>

### CODE REVIEW

File: public/data/Aniruddha-Inge.json [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+      "icon": "linkedin"
+    }
+  ],
+  "milestones": [
+    {
+      "title": "Started Contributing in Open Source",
+      "date": "December 2021",
+      "icon": "dollar",
+      "color": "grey",
+      "description": "Started my Open Source journey with EddieHub"
```
Comment (krishguptadev) [inline public/data/Aniruddha-Inge.json:34]:
```suggestion
      "description": "Started my Open Source journey with EddieHub!"
```

**Review by krishguptadev — APPROVED** [review]

**Review by emmalearnscode — APPROVED** [review]

**Review by nathgoutam93 — APPROVED** [review]

**Review by syedsohan — APPROVED** [review]

**Review by Pradumnasaraf — APPROVED** [review]

**Review by Pradumnasaraf — COMMENTED** [review]
Thanks for the contribution:D
Welcome to EddieHub and Open Source.

File: public/data/Aniruddha-Inge.json [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+      "url": "https://github.com/Aniruddha-Inge",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/AniruddhaInge",
+      "icon": "twitter"
+    },
+    {
+      "name": "Subscribe my YouTube channel",
```
Comment (Panquesito7) [inline public/data/Aniruddha-Inge.json:18]:
```suggestion
      "name": "Subscribe to my YouTube channel",
```

**Review by Panquesito7 — APPROVED** [review]
LGTM 🚀 Thank you for your contribution! 😄👍

**Comment by Aniruddha-Inge** [comment]
EddieHub Community is Awesome!!!! 😄

**Pull request merged**